sunpy/util/tests/test_util.py

```diff
@@ -121,3 +121,88 @@ def set_test6_n(self, *args, **kwargs):
             pass
 
     assert util.get_set_methods(A()) == {'test2', 'test6_n'}
+
+
+def test_notes_combined():
+    original_documentation = """
```

Comment:
eugh I hate it when things are dedented like this, would using `textwrap.dedent` defeat the object of the exercise?

Reply:
That's fair.  It's ugly as hell.  I'll fix it.

Reply:
Done (using `inspect.cleandoc()`)